packages/features/eventtypes/components/tabs/availability/EventAvailabilityTab.tsx

```diff
@@ -441,10 +441,10 @@ const TeamAvailability = ({
 
 const UseCommonScheduleSettingsToggle = ({ eventType, ...rest }: EventTypeScheduleProps) => {
   const { t } = useLocale();
-  const { setValue, resetField, getFieldState, getValues, watch } = useFormContext<FormValues>();
+  const { setValue, resetField, getValues, watch } = useFormContext<FormValues>();
 
   const [useHostSchedulesForTeamEvent, setUseHostSchedulesForTeamEvent] = useState(
-    !Boolean(getFieldState("schedule").isDirty ? getValues("schedule") : eventType.schedule)
+    !Boolean(eventType.schedule)
```

Comment:
For a team event, when the common schedule toggle is disabled, the scheduleId of the event is null, and when enabled, it’s an int. So let’s keep it simple and make the toggle check or the default state dependent only on the eventType scheduleId